docs/src/components/RunThunderID.tsx

```
@@ -0,0 +1,257 @@
+          '[data-theme="light"] &': {bgcolor: 'rgba(0,0,0,0.02)'},
+        }}
+      >
+        {TABS.map(({id, label, icon}) => {
+          const isActive = activeTab === id;
+          return (
+            <Box
+              key={id}
+              component="button"
+              onClick={() => setActiveTab(id)}
+              sx={{
+                alignItems: 'center',
```

Biome (2.5.1)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)